File: client/lib/services/data/department.dart
class Department {
  final int id;
  final String name, address, website;

  Department(this.id, this.name, this.address, this.website);

  Map<String, dynamic> toJson() => {
    'id': id,
    'name': name,
    'address': address,
    'website': website
  };
}

File: server/app/Models/Department.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Department extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'address',
        'phones',
        'website',
        'working_time',
        'coordinates',
        'last_update',
        'currency_info',
        'bank_name',
        'city'
    ];

    protected $casts = [
        'phones' => 'array',
        'coordinates' => 'array',
        'currency_info' => 'array',
        'created_at' => 'datetime:Y-m-d H:i:s',
        'updated_at' => 'datetime:Y-m-d H:i:s'
    ];
}
